lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.register = (server, options, next) => {
+  server.route([{
+    method: 'POST',
+    path: '/movies',
+    config: {
+      handler: (request, reply) => {
+        reply('hello world');
+      }
+    }
+  }]);
+  next();
```

Comment:
nit: add a newline above this line, and delete the extra newline below this line